tests/test_base.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+__author__ = "Mathematical Michael"
+__copyright__ = "Mathematical Michael"
+__license__ = "mit"
+
+
+def test_identity_mud_problem_1D(identity_problem_mud_1D):
+    # Arrange
+    D = identity_problem_mud_1D
+
+    # Act
+    mud_point = D.estimate()
+    updated_density = D._up
+    ratio = D._r
+
+    # Assert
+    assert np.round(mud_point, 1) == 0.5
+    assert np.sum(updated_density) > 0
+    assert np.mean(ratio) > 0
```

Comment:
@cdelcastillo21 I put this in as placeholders, I assumed both of these should be == 1 but they are hardly ever even close.

why may that be?

Reply:
`np.mean(ratio) == 1` is the same as checking `integral(updated_density) == 1` assuming that the samples are iid from initial (which they are for us).